flake.nix

```diff
@@ -29,6 +29,8 @@
           ${self.packages.${system}.themes}/lib/node_modules/$THEME/index.js  
       '';
 
+      generatePdf = pkgs.callPackage ./tools/pdf-export {};
```

Comment:
weren't you just telling me how callPackage does weird things and breaks the threaded creation of nix stuff? Something like that? remind me.

Reply:
It does weird things, but doesn't break threaded stuff, just makes finding things in reverse complicated. Bad for learning, good for organization.  Frustrating but necessary.